openedx/frontend-app-authoring
# Show spinner while loading library components [FC-0062]

## Description

Use `query.isLoading` (not `query.isFetching`) to show a spinner when loading Library components.

Prior to this change, the home page for a content library briefly shows "You haven't added any components to this library yet" while the library is loading.

This change affects Course Authors using v2 content libraries.

## Supporting information

Bug reported on Slack: [comment](https://openedx.slack.com/archives/C06JG6RFE4V/p1726775974983869)
Private-ref: [FAL-3859](https://tasks.opencraft.com/browse/FAL-3859)

## Testing instructions

1. Create a library with several components.
2. Force-refresh the library home page
    Note that a spinner is shown while the components are loading, and "You haven't added any components" is never shown.
3. Force-refresh the library components page.
    Note that a spinner is shown while the components are loading, and "You haven't added any components" is never shown.
